### PR TITLE
Add elf recipe

### PR DESCRIPTION
**Pull request by constantinpape**
Adds a recipe for elf, a python library for bio-medical image analysis

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/elf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "ed00f76b0880aed3943de4c66cc70a3946827f3662f85f8b2c5920bab23000ba" %}
+{% set name = "elf" %}
+
+package:
+    name: {{ name|lower }}
```
Comment (isuruf) [inline recipes/elf/meta.yaml:6]: `python-elf` here? `elf` means `Executable and Linkable Format` usually in packaging sense.

**Comment by constantinpape** [comment]
Thanks for reviewing, @isuruf. I changed the packet name and fixed the tests.
This is good to be merged from my side.

File: recipes/elf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{% set version = "0.3.0post0" %}
+{% set sha256 = "d57bd171b18dda4682870e8cac0faacb0d8a4c09b4564102c0ecff6b1612bf94" %}
+{% set name = "elf" %}
+
+package:
+    name: python-elf
```
Comment (isuruf) [inline recipes/elf/meta.yaml:6]: can you use 2 spaces instead of 4?

Comment (constantinpape) [reply recipes/elf/meta.yaml]: Done

**Comment by constantinpape** [comment]
@isuruf any more changes requested or can we get this merged?
Thanks!

**Comment by constantinpape** [comment]
Thanks for the fast review @isuruf!

**Pull request merged**